src/main/content/collections.html

```diff
@@ -44,6 +44,10 @@ <h1>{{t.instance.instances}}</h1>
                     </tr>
                 </thead>
                 <tbody id="collection-table-body"></tbody>
+                <div id="table-header-cli-version" class="hide">
+                    <p>CLI Version: </p>
+                    <p class="cli-version"></p>
```

Comment:
probably better as an ID since I would image there should only be 1 cli version on this page